egret/model_library/unit_commitment/power_balance.py

```diff
@@ -188,16 +188,11 @@ def _get_pg_expr_rule(t):
     def pg_expr_rule(block,b):
         m = block.model()
         # bus b, time t (S)
-        if m.storage_services:
-            return sum(m.PowerGenerated[g, t] for g in m.ThermalGeneratorsAtBus[b]) \
+        return sum(m.PowerGenerated[g, t] for g in m.ThermalGeneratorsAtBus[b]) \
```

Comment:
Comment to objective.py above applies here - seems like this isn't part of fuel supply modeling.